bin/copy-production-modules.js

```diff
@@ -121,7 +120,7 @@ function shipDependencies( pkgList ) {
 
 	debug( 'copying ' + pkgList.length + ' packages to ' + destDir );
 
-	mkdirp.sync( destDir );
+	fs.mkdirSync( destDir, { recursive: true } );
```

Comment:
Testing this locally, I don't think we actually need this, since this is already handled through `recursive-copy` (`rcopy`) below. ([ref](https://www.npmjs.com/package/recursive-copy#:~:text=Uses%20graceful%2Dfs%20and%20mkdirp%20to%20avoid%20filesystem%20errors))